2023-07-10 20:29:37,074 INFO    StreamThr :35961 [internal.py:wandb_internal():86] W&B internal server running at pid: 35961, started at: 2023-07-10 20:29:37.074162
2023-07-10 20:29:37,077 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status
2023-07-10 20:29:37,081 INFO    WriterThread:35961 [datastore.py:open_for_write():85] open: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_202937-muzvcap9/run-muzvcap9.wandb
2023-07-10 20:29:37,083 DEBUG   SenderThread:35961 [sender.py:send():369] send: header
2023-07-10 20:29:37,083 DEBUG   SenderThread:35961 [sender.py:send():369] send: run
2023-07-10 20:29:37,984 INFO    SenderThread:35961 [dir_watcher.py:__init__():211] watching files in: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_202937-muzvcap9/files
2023-07-10 20:29:37,984 INFO    SenderThread:35961 [sender.py:_start_run_threads():1100] run started: muzvcap9 with start time 1688992177.073442
2023-07-10 20:29:37,985 DEBUG   SenderThread:35961 [sender.py:send_request():396] send_request: summary_record
2023-07-10 20:29:37,985 INFO    SenderThread:35961 [sender.py:_save_file():1354] saving file wandb-summary.json with policy end
2023-07-10 20:29:37,989 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: check_version
2023-07-10 20:29:37,989 DEBUG   SenderThread:35961 [sender.py:send_request():396] send_request: check_version
2023-07-10 20:29:38,193 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: run_start
2023-07-10 20:29:38,202 DEBUG   HandlerThread:35961 [system_info.py:__init__():31] System info init
2023-07-10 20:29:38,202 DEBUG   HandlerThread:35961 [system_info.py:__init__():46] System info init done
2023-07-10 20:29:38,202 INFO    HandlerThread:35961 [system_monitor.py:start():181] Starting system monitor
2023-07-10 20:29:38,203 INFO    SystemMonitor:35961 [system_monitor.py:_start():145] Starting system asset monitoring threads
2023-07-10 20:29:38,204 INFO    HandlerThread:35961 [system_monitor.py:probe():201] Collecting system info
2023-07-10 20:29:38,206 INFO    SystemMonitor:35961 [interfaces.py:start():190] Started cpu monitoring
2023-07-10 20:29:38,207 INFO    SystemMonitor:35961 [interfaces.py:start():190] Started disk monitoring
2023-07-10 20:29:38,209 INFO    SystemMonitor:35961 [interfaces.py:start():190] Started gpu monitoring
2023-07-10 20:29:38,210 INFO    SystemMonitor:35961 [interfaces.py:start():190] Started memory monitoring
2023-07-10 20:29:38,211 INFO    SystemMonitor:35961 [interfaces.py:start():190] Started network monitoring
2023-07-10 20:29:38,225 DEBUG   HandlerThread:35961 [system_info.py:probe():195] Probing system
2023-07-10 20:29:38,230 DEBUG   HandlerThread:35961 [gitlib.py:_init_repo():53] git repository is invalid
2023-07-10 20:29:38,230 DEBUG   HandlerThread:35961 [system_info.py:probe():240] Probing system done
2023-07-10 20:29:38,230 DEBUG   HandlerThread:35961 [system_monitor.py:probe():210] {'os': 'Linux-5.19.0-46-generic-x86_64-with-glibc2.35', 'python': '3.9.16', 'heartbeatAt': '2023-07-10T12:29:38.225932', 'startedAt': '2023-07-10T12:29:37.065129', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '/home/zhuhe/HPE-with-Diffusion/main.py', 'codePath': 'main.py', 'host': '415-2', 'username': 'zhuhe', 'executable': '/home/zhuhe/.conda/envs/diffhpe/bin/python', 'cpu_count': 16, 'cpu_count_logical': 24, 'cpu_freq': {'current': 2875.147125, 'min': 800.0, 'max': 4783.333333333333}, 'cpu_freq_per_core': [{'current': 2360.894, 'min': 800.0, 'max': 5100.0}, {'current': 2989.075, 'min': 800.0, 'max': 5100.0}, {'current': 2305.145, 'min': 800.0, 'max': 5100.0}, {'current': 2901.97, 'min': 800.0, 'max': 5100.0}, {'current': 2661.259, 'min': 800.0, 'max': 5100.0}, {'current': 2100.0, 'min': 800.0, 'max': 5100.0}, {'current': 2678.661, 'min': 800.0, 'max': 5100.0}, {'current': 2100.0, 'min': 800.0, 'max': 5100.0}, {'current': 2625.836, 'min': 800.0, 'max': 5200.0}, {'current': 2647.131, 'min': 800.0, 'max': 5200.0}, {'current': 2646.745, 'min': 800.0, 'max': 5200.0}, {'current': 2100.0, 'min': 800.0, 'max': 5200.0}, {'current': 2649.93, 'min': 800.0, 'max': 5100.0}, {'current': 2626.068, 'min': 800.0, 'max': 5100.0}, {'current': 3137.744, 'min': 800.0, 'max': 5100.0}, {'current': 3810.83, 'min': 800.0, 'max': 5100.0}, {'current': 1679.722, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}], 'disk': {'total': 93.31179428100586, 'used': 0.018383026123046875}, 'gpu': 'NVIDIA GeForce RTX 3090', 'gpu_count': 2, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3090', 'memory_total': 25769803776}, {'name': 'NVIDIA GeForce RTX 3090', 'memory_total': 25769803776}], 'memory': {'total': 31.185558319091797}}
2023-07-10 20:29:38,230 INFO    HandlerThread:35961 [system_monitor.py:probe():211] Finished collecting system info
2023-07-10 20:29:38,230 INFO    HandlerThread:35961 [system_monitor.py:probe():214] Publishing system info
2023-07-10 20:29:38,230 DEBUG   HandlerThread:35961 [system_info.py:_save_pip():51] Saving list of pip packages installed into the current environment
2023-07-10 20:29:38,231 DEBUG   HandlerThread:35961 [system_info.py:_save_pip():67] Saving pip packages done
2023-07-10 20:29:38,231 DEBUG   HandlerThread:35961 [system_info.py:_save_conda():74] Saving list of conda packages installed into the current environment
2023-07-10 20:29:38,987 INFO    Thread-17 :35961 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_202937-muzvcap9/files/requirements.txt
2023-07-10 20:29:38,987 INFO    Thread-17 :35961 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_202937-muzvcap9/files/conda-environment.yaml
2023-07-10 20:29:38,987 INFO    Thread-17 :35961 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_202937-muzvcap9/files/wandb-summary.json
2023-07-10 20:29:39,103 DEBUG   HandlerThread:35961 [system_info.py:_save_conda():86] Saving conda packages done
2023-07-10 20:29:39,104 INFO    HandlerThread:35961 [system_monitor.py:probe():216] Finished publishing system info
2023-07-10 20:29:39,107 DEBUG   SenderThread:35961 [sender.py:send():369] send: files
2023-07-10 20:29:39,107 INFO    SenderThread:35961 [sender.py:_save_file():1354] saving file wandb-metadata.json with policy now
2023-07-10 20:29:39,114 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 20:29:39,114 DEBUG   SenderThread:35961 [sender.py:send_request():396] send_request: stop_status
2023-07-10 20:29:39,571 ERROR   wandb-upload_0:35961 [internal_api.py:upload_file():2192] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ada4321/DiffHPE/muzvcap9/wandb-metadata.json?Expires=1689078579&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=QoFXEzh92VgVMp%2BJp03msBIzaIVQob%2FCHTceW%2Fguec7k71T5gUwdnPN15JziYzWj7CJTud3SqemSTcwxEFUIO5bQkPbW7d%2BqiiKNgBV9YiE6gzUAaLDq%2Fs6MgiKqFPkQvFeXydib9nu4x58V7X8nVuClsXLRQx6JSH0Q9SJ3cYANg6yMDSNUHGp6Ukh7%2BcVxkzW0H4%2BmaX%2Bny9uJjNR6HAaHH0la%2FjW326fXN1exb%2BJZYC%2BCvk6jIWJwkl30yIzbughV5oAnOoqmJw39dIMTdmEGQyASiJzJTPYAVqQSuyUqjETYm4MhLrQ7fbiA0ML97qgLFqrGGaf8cGTUBOsWDg%3D%3D: 404 Client Error: Not Found for url: https://storage.googleapis.com/wandb-production.appspot.com/ada4321/DiffHPE/muzvcap9/wandb-metadata.json?Expires=1689078579&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=QoFXEzh92VgVMp%2BJp03msBIzaIVQob%2FCHTceW%2Fguec7k71T5gUwdnPN15JziYzWj7CJTud3SqemSTcwxEFUIO5bQkPbW7d%2BqiiKNgBV9YiE6gzUAaLDq%2Fs6MgiKqFPkQvFeXydib9nu4x58V7X8nVuClsXLRQx6JSH0Q9SJ3cYANg6yMDSNUHGp6Ukh7%2BcVxkzW0H4%2BmaX%2Bny9uJjNR6HAaHH0la%2FjW326fXN1exb%2BJZYC%2BCvk6jIWJwkl30yIzbughV5oAnOoqmJw39dIMTdmEGQyASiJzJTPYAVqQSuyUqjETYm4MhLrQ7fbiA0ML97qgLFqrGGaf8cGTUBOsWDg%3D%3D
2023-07-10 20:29:39,572 ERROR   wandb-upload_0:35961 [internal_api.py:upload_file():2194] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3624'}
2023-07-10 20:29:39,572 ERROR   wandb-upload_0:35961 [internal_api.py:upload_file():2196] upload_file response body: b'<!DOCTYPE html>\n<html lang=en>\n  <meta charset=utf-8>\n  <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">\n  <title>Error 404 (Not Found)!!1</title>\n  <style>\n    *{margin:0;padding:0}html,code{font:15px/22px arial,sans-serif}html{background:#fff;color:#222;padding:15px}body{margin:7% auto 0;max-width:390px;min-height:180px;padding:30px 0 15px}* > body{background:url(//www.google.com/images/errors/robot.png) 100% 5px no-repeat;padding-right:205px}p{margin:11px 0 22px;overflow:hidden}ins{color:#777;text-decoration:none}a img{border:0}@media screen and (max-width:772px){body{background:none;margin-top:0;max-width:none;padding-right:0}}#logo{background:url(//www.google.com/images/branding/googlelogo/1x/googlelogo_color_150x54dp.png) no-repeat;margin-left:-5px}@media only screen and (min-resolution:192dpi){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat 0% 0%/100% 100%;-moz-border-image:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) 0}}@media only screen and (-webkit-min-device-pixel-ratio:2){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat;-webkit-background-size:100% 100%}}#logo{display:inline-block;height:54px;width:150px}\n  </style>\n  <a href=//www.google.com/><span id=logo aria-label=Google></span></a>\n  <p><b>404.</b> <ins>That\xe2\x80\x99s an error.</ins>\n  <p>The requested URL <code>/wandb-production.appspot.com/ada4321/DiffHPE/muzvcap9/wandb-metadata.json</code> was not found on this server.  <ins>That\xe2\x80\x99s all we know.</ins>\n'
2023-07-10 20:29:39,692 DEBUG   SenderThread:35961 [sender.py:send():369] send: telemetry
2023-07-10 20:29:39,987 INFO    Thread-17 :35961 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_202937-muzvcap9/files/conda-environment.yaml
2023-07-10 20:29:39,987 INFO    Thread-17 :35961 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_202937-muzvcap9/files/output.log
2023-07-10 20:29:39,988 INFO    Thread-17 :35961 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_202937-muzvcap9/files/wandb-metadata.json
2023-07-10 20:29:41,577 ERROR   wandb-upload_0:35961 [upload_job.py:push():136] Failed to upload file: /tmp/tmp5kj_cofqwandb/yoslqxt7-wandb-metadata.json
Traceback (most recent call last):
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 212, in wrapped_fn
    return retrier(*args, **kargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2218, in upload_file
    wandb._sentry.reraise(e)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/analytics/sentry.py", line 146, in reraise
    raise exc.with_traceback(sys.exc_info()[2])
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2190, in upload_file
    response.raise_for_status()
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://storage.googleapis.com/wandb-production.appspot.com/ada4321/DiffHPE/muzvcap9/wandb-metadata.json?Expires=1689078579&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=QoFXEzh92VgVMp%2BJp03msBIzaIVQob%2FCHTceW%2Fguec7k71T5gUwdnPN15JziYzWj7CJTud3SqemSTcwxEFUIO5bQkPbW7d%2BqiiKNgBV9YiE6gzUAaLDq%2Fs6MgiKqFPkQvFeXydib9nu4x58V7X8nVuClsXLRQx6JSH0Q9SJ3cYANg6yMDSNUHGp6Ukh7%2BcVxkzW0H4%2BmaX%2Bny9uJjNR6HAaHH0la%2FjW326fXN1exb%2BJZYC%2BCvk6jIWJwkl30yIzbughV5oAnOoqmJw39dIMTdmEGQyASiJzJTPYAVqQSuyUqjETYm4MhLrQ7fbiA0ML97qgLFqrGGaf8cGTUBOsWDg%3D%3D

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/upload_job.py", line 127, in push
    self._api.upload_file_retry(
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/apis/normalize.py", line 51, in wrapper
    raise CommError(message, error)
wandb.errors.CommError: <Response [404]>
2023-07-10 20:29:41,986 INFO    Thread-17 :35961 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_202937-muzvcap9/files/output.log
2023-07-10 20:29:42,350 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:29:43,584 ERROR   Thread-16 :35961 [step_upload.py:_handle_event():200] Failed to upload file: /tmp/tmp5kj_cofqwandb/yoslqxt7-wandb-metadata.json
Traceback (most recent call last):
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 212, in wrapped_fn
    return retrier(*args, **kargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2218, in upload_file
    wandb._sentry.reraise(e)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/analytics/sentry.py", line 146, in reraise
    raise exc.with_traceback(sys.exc_info()[2])
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2190, in upload_file
    response.raise_for_status()
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://storage.googleapis.com/wandb-production.appspot.com/ada4321/DiffHPE/muzvcap9/wandb-metadata.json?Expires=1689078579&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=QoFXEzh92VgVMp%2BJp03msBIzaIVQob%2FCHTceW%2Fguec7k71T5gUwdnPN15JziYzWj7CJTud3SqemSTcwxEFUIO5bQkPbW7d%2BqiiKNgBV9YiE6gzUAaLDq%2Fs6MgiKqFPkQvFeXydib9nu4x58V7X8nVuClsXLRQx6JSH0Q9SJ3cYANg6yMDSNUHGp6Ukh7%2BcVxkzW0H4%2BmaX%2Bny9uJjNR6HAaHH0la%2FjW326fXN1exb%2BJZYC%2BCvk6jIWJwkl30yIzbughV5oAnOoqmJw39dIMTdmEGQyASiJzJTPYAVqQSuyUqjETYm4MhLrQ7fbiA0ML97qgLFqrGGaf8cGTUBOsWDg%3D%3D

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/step_upload.py", line 288, in run_and_notify
    self._do_upload_sync(event)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/step_upload.py", line 330, in _do_upload_sync
    job.run()
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/upload_job.py", line 57, in run
    self.push()
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/upload_job.py", line 127, in push
    self._api.upload_file_retry(
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/apis/normalize.py", line 51, in wrapper
    raise CommError(message, error)
wandb.errors.CommError: <Response [404]>
2023-07-10 20:29:43,987 INFO    Thread-17 :35961 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_202937-muzvcap9/files/output.log
2023-07-10 20:29:47,554 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:29:49,990 INFO    Thread-17 :35961 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_202937-muzvcap9/files/output.log
2023-07-10 20:29:51,991 INFO    Thread-17 :35961 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_202937-muzvcap9/files/output.log
2023-07-10 20:29:52,566 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:29:54,115 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 20:29:54,117 DEBUG   SenderThread:35961 [sender.py:send_request():396] send_request: stop_status
2023-07-10 20:29:58,352 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:30:03,354 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:30:08,360 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:30:09,004 INFO    Thread-17 :35961 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_202937-muzvcap9/files/config.yaml
2023-07-10 20:30:09,114 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 20:30:09,116 DEBUG   SenderThread:35961 [sender.py:send_request():396] send_request: stop_status
2023-07-10 20:30:14,342 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:30:19,343 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:30:21,140 DEBUG   SenderThread:35961 [sender.py:send():369] send: telemetry
2023-07-10 20:30:22,015 INFO    Thread-17 :35961 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_202937-muzvcap9/files/output.log
2023-07-10 20:30:24,114 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 20:30:24,114 DEBUG   SenderThread:35961 [sender.py:send_request():396] send_request: stop_status
2023-07-10 20:30:25,332 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:30:27,462 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: partial_history
2023-07-10 20:30:28,018 INFO    Thread-17 :35961 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_202937-muzvcap9/files/output.log
2023-07-10 20:30:30,020 INFO    Thread-17 :35961 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_202937-muzvcap9/files/output.log
2023-07-10 20:30:30,464 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:30:35,466 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:30:38,212 DEBUG   SystemMonitor:35961 [system_monitor.py:_start():159] Starting system metrics aggregation loop
2023-07-10 20:30:38,217 DEBUG   SenderThread:35961 [sender.py:send():369] send: stats
2023-07-10 20:30:39,117 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 20:30:39,119 DEBUG   SenderThread:35961 [sender.py:send_request():396] send_request: stop_status
2023-07-10 20:30:41,358 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:30:42,029 INFO    Thread-17 :35961 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_202937-muzvcap9/files/config.yaml
2023-07-10 20:30:46,656 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:30:51,658 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:30:54,118 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 20:30:54,120 DEBUG   SenderThread:35961 [sender.py:send_request():396] send_request: stop_status
2023-07-10 20:30:57,353 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:31:02,354 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:31:07,355 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:31:08,220 DEBUG   SenderThread:35961 [sender.py:send():369] send: stats
2023-07-10 20:31:09,117 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 20:31:09,118 DEBUG   SenderThread:35961 [sender.py:send_request():396] send_request: stop_status
2023-07-10 20:31:12,371 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:31:17,372 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:31:22,374 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:31:24,117 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 20:31:24,119 DEBUG   SenderThread:35961 [sender.py:send_request():396] send_request: stop_status
2023-07-10 20:31:28,351 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:31:33,355 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:31:38,226 DEBUG   SenderThread:35961 [sender.py:send():369] send: stats
2023-07-10 20:31:39,120 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 20:31:39,122 DEBUG   SenderThread:35961 [sender.py:send_request():396] send_request: stop_status
2023-07-10 20:31:39,377 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:31:44,378 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:31:49,380 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:31:54,118 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 20:31:54,119 DEBUG   SenderThread:35961 [sender.py:send_request():396] send_request: stop_status
2023-07-10 20:31:55,345 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:32:00,347 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:32:05,348 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:32:08,228 DEBUG   SenderThread:35961 [sender.py:send():369] send: stats
2023-07-10 20:32:09,118 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 20:32:09,119 DEBUG   SenderThread:35961 [sender.py:send_request():396] send_request: stop_status
2023-07-10 20:32:10,355 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:32:15,356 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:32:20,358 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:32:24,118 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 20:32:24,118 DEBUG   SenderThread:35961 [sender.py:send_request():396] send_request: stop_status
2023-07-10 20:32:26,340 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:32:31,341 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 20:32:36,343 DEBUG   HandlerThread:35961 [handler.py:handle_request():144] handle_request: status_report
